src/pages/trader-tools/_tools.js

```diff
@@ -0,0 +1,164 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import { graphql, useStaticQuery } from 'gatsby'
+import device from 'themes/device'
+import { Container, SectionContainer, Flex } from 'components/containers'
+import { Header, Text, QueryImage } from 'components/elements'
+import { LinkButton } from 'components/form'
+
+const StyledSection = styled(SectionContainer)`
+    @media ${device.tabletL} {
+        padding: 40px 0;
+    }
+`
+const ImageWrapper = styled.div`
+    width: 100%;
+    height: 100%;
+    margin-right: ${(props) => props.margin_right};
+    margin-left: ${(props) => props.margin_left};
+
+    @media ${device.tabletL} {
+        margin: 24px 0 0;
+    }
+`
+
+const Content = styled(Flex)`
+    max-width: 39rem;
+    margin-right: ${(props) => props.margin_right};
+    margin-left: ${(props) => props.margin_left};
+`
+const StyledLinkButton = styled(LinkButton)`
+    margin-top: 16px;
+    width: fit-content;
+
+    &:hover {
+        cursor: pointer;
+    }
+
+    @media ${device.tabletL} {
+        margin-top: 32px;
+        width: 100%;
+    }
+`
+const Divider = styled.div`
+    :not(:last-child) {
+        height: 1px;
+        width: 100%;
+        background: var(--color-grey-21);
+        margin: 80px 0;
+    }
+
+    @media ${device.tabletL} {
+        :not(:last-child) {
+            margin: 40px 0;
+        }
+    }
+`
+const DesktopWrapper = styled(Flex)`
+    flex-direction: ${(props) => props.flex_direction};
+    align-items: space-between;
+
+    @media ${device.tabletL} {
+        display: none;
+    }
+`
+const MobileWrapper = styled(Flex)`
+    display: none;
+    @media ${device.tabletL} {
+        display: block;
+        max-width: 500px;
+    }
+`
+
+const query = graphql`
+    query {
+        margin: file(relativePath: { eq: "trade-tools/landing/margin-calculator.png" }) {
+            ...fadeIn
+        }
+        swap: file(relativePath: { eq: "trade-tools/landing/swap-calculator.png" }) {
+            ...fadeIn
+        }
+        pip: file(relativePath: { eq: "trade-tools/landing/pip-calculator.png" }) {
+            ...fadeIn
+        }
+        pnl_margin: file(relativePath: { eq: "trade-tools/landing/pnl-margin.png" }) {
+            ...fadeIn
+        }
+        pnl_multipliers: file(relativePath: { eq: "trade-tools/landing/pnl-multipliers.png" }) {
+            ...fadeIn
+        }
+    }
+`
+
+const TradingTools = ({ tools, reverse }) => {
+    const data = useStaticQuery(query)
+    return (
+        <StyledSection background="white">
+            <Container fd="column">
+                {tools.map((item, index) => {
+                    let is_even = reverse ? (index + 1) % 2 : index % 2
+                    return (
+                        <>
+                            <DesktopWrapper
```

Comment:
There's a concern for returning a `DesktopWrapper` & `MobileWrapper` here for every item being mapped. But due to the layout of the desktop vs mobile design, this was what I came up with. Open to any suggestions of restructuring this! 😄

Reply:
@mustofa-binary @yashim-deriv @kevinw-binary @suthesan-deriv @njazuli-deriv

Reply:
The images are the same for both mobile and desktop?My suggestion is maybe can apply 'show and hide' just for the button.